### DISCUSSION TITLE
Relaxation of ESMfold results by openmm

### DISCUSSION THREAD

**Discussion by xvtyzn** [facebookresearch/esm#415] [Q&A] [answered]
Hi.

I am getting more and more excited everyday about the great results of ESMfold.
I have a question about ESMfold's relaxation. It is stated in ESMatlas that ESMfold does not currently perform relaxation.
https://esmatlas.com/about

> There are a few things to keep in mind when using our resource. All the structures we provide are predictions and, especially in regions with low pLDDT, ESMFold structures may differ from those that are determined via experimental methods. We also do not apply physics-based refinement or relaxation of our structures. 

However, it seems that openmm is specified as a dependency in the esmfold execution environment. Are there plans to implement relaxation by openmm in the future?

```
name: esmfold
channels:
  - conda-forge
  - bioconda
  - pytorch
dependencies:
  - conda-forge::openmm=7.5.1
```

Best,

**Comment by tomsercu** [comment, marked as answer]
Thanks for the catch! We can probably just clear openmm from that environment. We're not planning to add openmm relaxation to our codebase.
But of course end users can do the relaxation for any structure in the Atlas or predicted by ESMFold. We didn't see much gains from the relaxation for our purposes, eg structure prediction performance.
